samples/tfm/tfm_hello_world/src/main.c

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA.
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <sys/printk.h>
+#include <string.h>
+#include <stdio.h>
+#include <tfm_ns_interface.h>
+#include "psa/crypto.h"
+
+void main(void)
+{
+	#define HELLO_PATTERN "Hello World! %s\n"
```

Comment:
guess the #define can start from the beginning of the line

Reply:
Done